subsys/nfc/lib/platform.c

```diff
@@ -35,6 +35,18 @@ LOG_MODULE_REGISTER(nfc_platform, CONFIG_NFC_PLATFORM_LOG_LEVEL);
 static struct onoff_manager *hf_mgr;
 static struct onoff_client cli;
 
+ISR_DIRECT_DECLARE(nfc_isr_wrapper)
+{
+	nrfx_nfct_irq_handler();
+
+	ISR_DIRECT_PM();
```

Comment:
Why do we exit power management idle mode at this point? For instance, why shouldn't we do it before the nrfx_nfct_irq_handler call?

Reply:
We do this for better interrupt latency. Without this all the hardware will be resumed from low-power state before the ISR is executed, which can be very time-consuming.